client/app/admin/admin.component.html

```diff
@@ -0,0 +1,148 @@
+<div class="container-fluid fill-height p-0">
+
+  <div class="row half">
+    <div id="menuBar" class="col-3 sidebar pl-0">
+      <button class="savebtn" (click)="saveMenu()">Save</button>
+      <button class="closebtn" (click)="closeMenuNav()">Edit Menu -</button>
+      <div class="sideScroll">
+        <table class="table table-hover table-sm" width="100%">
+          <thead>
+            <tr>
+              <th>Item</th>
+              <th>Price</th>
+              <th>Category</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr *ngFor="let item of menu">
+              <td contenteditable="true">{{ item.name }}</td>
```

Comment:
I think these will need to be changed to form inputs in order to bind correctly to the data, otherwise it will just modify the display but not the prop value on the `item`.